### PR TITLE
docs: improved en version of sql/select_overview

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
fixed some foramts, like tahe title of table
finish translation 
updated the table

### CODE REVIEW

File: docs/en/reference/sql/dql/SELECT_STATEMENT.md [hunk 1]
```diff
@@ -111,27 +111,21 @@ TableAsName
...
+| `ORDER BY` Clause                              |              |                     |                     | Standard SQL also supports the OrderBy clause, however OpenMLDB does not support this clause currently. For example, the query `SELECT * from t1 ORDER BY col1;` is not supported in OpenMLDB.                                                                                                                                                                                          |
 
 ```{warning}
-The online mode or the stand-alone version of the selection may not obtain complete data.
-Because a query may perform a large number of scans on multiple tablet servers, for the stability of tablet servers, a single tablet server limits the maximum amount of scanned data, namely `scan_max_bytes_size`.
+The `SELECT` running in online mode or the stand-alone version may not obtain complete data.
+Because a query may perform a large number of scans on multiple tablets, for stability, the maximum scanned data on one tablet has been limited, namely `scan_max_bytes_size`.
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_STATEMENT.md]: the maximum scanned data on one tablet has been limited
the largest number of bytes to scan is limited

File: docs/en/reference/sql/dql/SELECT_STATEMENT.md [hunk 2]
```diff
@@ -111,27 +111,21 @@ TableAsName
...
 
-If the select result is truncated, the tablet server will display a log of `reach the max byte ...`, but the query will not report an error.
+If the select results are truncated, the message of `reach the max byte ...` will be recorded in the tablet's log, but there will not be an error.
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_STATEMENT.md]: but there will not be an error
but there will be no error

File: docs/en/reference/sql/dql/SELECT_STATEMENT.md [hunk 3]
```diff
@@ -111,27 +111,21 @@ TableAsName
...
+| [`SELECT Clause`](#selectexprlist)             | **``✓``**    | **``✓``**           | **``✓``**           | A list of projection operations, generally including column names, expressions, or ‘*’ for all columns                                                                                                                                                                                                                                                                                  |
+| [`FROM Clause`](#tablerefs)                    | **``✓``**    | **``✓``**           | **``✓``**           | The From clause indicates the data source.<br />The data source can be one table (`select * from t;`) or multiple tables that JOIN together (`select * from t1 join t2;`) or no table ( `select 1+1;`), see [NO_TABLE SELECT](../dql/NO_TABLE_SELECT_CLAUSE.md)                                                                                                                         |
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_STATEMENT.md]: From => FROM

Comment (lumianph) [reply docs/en/reference/sql/dql/SELECT_STATEMENT.md]: or multiple tables that JOIN together (`select * from t1 join t2;`)
=>
can you check this statement is supported?

Comment (lumianph) [reply docs/en/reference/sql/dql/SELECT_STATEMENT.md]: capitalize all the SQL keyword, WHERE, GROUP BY, HAVING, WINDOW, ... which are shown in this table

File: docs/en/reference/sql/dql/SELECT_STATEMENT.md [hunk 4]
```diff
@@ -111,27 +111,21 @@ TableAsName
...
+| [`LIMIT` Clause](../dql/LIMIT_CLAUSE.md)       | **``✓``**    | **``✓``**           | **``✓``**           | The Limit clause is used to limit the number of results. OpenMLDB currently only supports one parameter to limit the maximum number of rows of returned data.                                                                                                                                                                                                                           |
+| `ORDER BY` Clause                              |              |                     |                     | Standard SQL also supports the OrderBy clause, however OpenMLDB does not support this clause currently. For example, the query `SELECT * from t1 ORDER BY col1;` is not supported in OpenMLDB.                                                                                                                                                                                          |
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_STATEMENT.md]: OrderBy => ORDER BY

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/reference/sql/dql/SELECT_STATEMENT.md (hunk 3 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/SELECT_STATEMENT.md]: > or multiple tables that JOIN together (`select * from t1 join t2;`) => can you check this statement is supported?

it is not supported.

is it ok if I change it to 
 or multiple tables that `LAST JOIN` together (see [JOIN CLAUSE](https://openmldb.ai/docs/en/main/reference/sql/dql/JOIN_CLAUSE.html))

**Review by lumianph — APPROVED** [review]

**Pull request merged**